src/apps/answers/filters.py

```diff
@@ -34,6 +34,7 @@ class AppletSubmissionsFilter(BaseQueryParams):
 class AppletSubmitDateFilter(BaseQueryParams):
     respondent_id: uuid.UUID | None
     target_subject_id: uuid.UUID | None
+    activity_id: str | None
```

Comment:
I'm just [checking in](https://mindlogger.atlassian.net/browse/M2-8215?focusedCommentId=26747) with @ChaconC about task definition. I think this endpoint should also accept an optional `activity_flow_id`. Stay tuned!

Reply:
Or maybe use a field named `activity_or_flow_id` and in query use same code we are using in other endpoints

```
                case(
                    (
                        AnswerSchema.flow_history_id.isnot(None),
                        AnswerSchema.id_from_history_id(AnswerSchema.flow_history_id) == str(activity_or_flow_id),
                    ),
                    else_=AnswerSchema.id_from_history_id(AnswerSchema.activity_history_id) == str(activity_or_flow_id),
                )
```